filedialogs/filedialogs.py

```diff
@@ -50,7 +50,9 @@ def open_file_dialog(
     if ext is None:
         ext = "All Files\0*.*\0"
     else:
-        ext = "".join([f"{name}\0*.{extension}\0" for name, extension in ext])
+        extFilter = ""
+        for name, extensions in ext:
+            extFilter += f"{name}\0" + ";".join(f"*.{extension}" for extension in extensions) + "\0"
```

Comment:
This does not work anymore if I want to use just a single file extension per description, e.g., `ext=[("python": "py")]`, and I have to do `ext=[("python": ("py",))]`. I think we should keep this option while allowing tuples of file extentions.